flash_english_backend/src/Controllers/PingController.php

```
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Controllers;
+
+class PingController
+{
+	private string $logFile = __DIR__ . "/../../public/debug.log";
```

PHPMD (2.15.0)

[warning] Avoid unused private fields such as '$logFile'. (undefined)

(UnusedPrivateField)